pkg/apis/elasticsearch/v1/validations.go

```diff
@@ -28,6 +30,7 @@ const (
 	nodeRolesInOldVersionMsg = "node.roles setting is not available in this version of Elasticsearch"
 	parseStoredVersionErrMsg = "Cannot parse current Elasticsearch version. String format must be {major}.{minor}.{patch}[-{label}]"
 	parseVersionErrMsg       = "Cannot parse Elasticsearch version. String format must be {major}.{minor}.{patch}[-{label}]"
+	pvcImmutableMsg          = "Volume claim templates cannot be modified. Only storage requests may be increased, if the storage class allows volume expansion."
```

Comment:
```suggestion
	pvcImmutableMsg          = "Volume claim templates can only have their storage request increased (if the storage class allows volume expansion), any other modifications are forbidden."
```

Or some other rewording - right now the first sentence is contrary to the second one :)